catalyst/src/logs.js

```diff
@@ -0,0 +1,80 @@
+function initializeLogs(sandbox) {
+    // Uses console.info() because VBG blocks console.log();
+
+    const environmentLogDefaults = {
+        // VCG
+        b02d16aa80: 'silent', // Prod
+        add8459f1c: 'normal', // Staging
+        '55e68a2ba9': 'normal', // Development
```

Comment:
Why are some quoted and some aren't?

Reply:
That's prettier automatic formatting. For JS they only _have_ to be quoted if they start with a number. It bothered me too, but configure it I'd have to install ESLint too and I didn't feel like it